application/core/MY_Model.php

```diff
@@ -146,6 +146,8 @@ public function paginate($base_url, $offset = 0, $uri_segment = 3)
 
         $this->set_defaults();
         $this->run_filters();
+		
+		$this->current_page = ($offset / $per_page) + 1;
```

Comment:
Could you correct your indentation for all files? :)

Reply:
ready